sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/BlobServiceVersion.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.storage.blob;
+
+
+/**
+ * The versions of Azure Storage Blob supported by this client library.
+ */
+public enum BlobServiceVersion {
```

Comment:
Will we have a core ServiceVersion type this can inherit from like in Java?

Reply:
I think that will be the case. What do you think @anuchandy, @bsiegel?